packages/client/src/app/pages/portfolio/works/works.component.ts

```diff
@@ -46,8 +45,8 @@ export class WorksComponent implements OnInit {
 
         this.route.data.subscribe(data => {
```

Comment:
While you're changing this file anyway, can you get away with changing this `route.data.subscribe` to `route.data.snapshot`?